src/librustdoc/clean/mod.rs

```diff
@@ -2430,11 +2430,11 @@ pub struct ViewListIdent {
 impl Clean<ViewListIdent> for ast::PathListItem {
     fn clean(&self, cx: &DocContext) -> ViewListIdent {
         match self.node {
-            ast::PathListIdent { id, name } => ViewListIdent {
+            ast::PathListIdent { id, name, .. } => ViewListIdent {
```

Comment:
I think these cases need to be handled to show reexports correctly, this will probably need to update the `ViewListIdent` structures here, and can you be sure to add a rustdoc test for this?